### PR TITLE
Import fenix quarantine

### PR DESCRIPTION
**Pull request by mozilla-pontoon**
Import fenix quarantine

### CODE REVIEW

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1376,6 +1378,23 @@
     <!-- Text shown in the notification that pops up to remind the user that a private browsing session is active. -->
     <string name="notification_pbm_delete_text_2">Close private tabs</string>
 
+    <!-- Microsuverys -->
+    <!-- Text shown in prompt for printing microsurvey. -->
+    <string name="microsurvey_prompt_printing_title" tools:ignore="UnusedResources">Help make printing in Firefox better. It only takes a sec</string>
```
Comment (Delphine) [inline mozilla-mobile/fenix/app/src/main/res/values/strings.xml:1383]: @Amejia481 Could you give a brief comment as to what "sec" means, for localizers please?

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -661,6 +661,8 @@
     <string name="addons_permissions_allow_for_all_sites" tools:ignore="UnusedResources">Allow for all sites</string>
     <!-- The subtitle for the allow for all sites preference toggle -->
     <string name="addons_permissions_allow_for_all_sites_subtitle" tools:ignore="UnusedResources">If you trust this extension, you can give it permission on every website.</string>
+    <!-- The text shown when an add-on does not require permissions -->
```
Comment (Delphine) [inline mozilla-mobile/fenix/app/src/main/res/values/strings.xml:664]: Nit: "add-on" is "extension" now

**Review by Delphine — CHANGES_REQUESTED** [review]

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Amejia481) [reply mozilla-mobile/fenix/app/src/main/res/values/strings.xml:1383]: Sorry for the delay I filed https://phabricator.services.mozilla.com/D212175 to add the suggestions.

**Comment by Amejia481** [comment]
As https://phabricator.services.mozilla.com/D212175 landed we could close this ticket :)

**Pull request merged**